### ISSUE TITLE
Empty Footer

### ISSUE THREAD

**Issue by Alecaddd** [thundernest/thunderblog#25]
The Footer area should have custom menus filled with our social media handles, various links, and the usual "Mozilla copyright" disclaimer, pretty much like we do in the main website.

This should most likely be possible to modify from the WordPress Admin dashboard, unless there's no menu location created for that footer area.

@devmount, is that section already customizable with a menu?

**Comment by devmount** [comment]
Yes, that is already implemented. As described in the [readme](https://github.com/thundernest/thunderblog#menus) of this repo, simply add a menu on the footer slot (*Appearance > Menus*) and up to 3 widgets for the footer area containing a h4 with an unordered list following (*Appearance > Widgets*).

**Comment by devmount** [comment]
@Alecaddd I saw that the footer is still empty. Do you need more information to fill the footer area with content?

**Comment by Alecaddd** [comment]
No, no worries. 
We just didn't find the time and the right content to put there.

**Comment by devmount** [comment]
Alright. Just for reference and some inspiration, this was the footer layout from the mockup:

![image](https://user-images.githubusercontent.com/5441654/204364870-b384d7f1-27c2-4bfe-90f1-29de50378bdb.png)

**Comment by devmount** [comment]
@Alecaddd Any news on this? People are noticing the empty footer (#57)

**Comment by MelissaAutumn** [comment]
Suggestion: What if we were to mimic the new thunderbird.net footer and make it seem like a more integrated experience:
https://stage.thunderbird.net/en-CA/

**Comment by devmount** [comment]
That is a great suggestion, Melissa! The Theme should already support this. You can add a menu on the footer slot (*Appearance > Menus*) which appears on the very bottom of the page and up to 3 widgets each for one footer column containing a h4 with an unordered list following (*Appearance > Widgets*).
